metastats-application/src/main/java/life/qbic/metastats/cli/MetaStatsEntryPoint.java

```diff
@@ -21,6 +21,11 @@
     public static void main(final String[] args) {
         LOG.debug("Starting MetaStats tool");
         final ToolExecutor executor = new ToolExecutor();
-        executor.invoke(MetaStatsTool.class, MetaStatsCommand.class, args);
+
+        if(args.length != 0){
```

Comment:
```suggestion
        if(args.length == 2) {
```
If I understand your documentation correctly you always require `-c` and `-p` parameters to be present.

Reply:
1146e57